generators/app/index.js

```diff
@@ -143,6 +143,9 @@ module.exports = class extends Generator {
 
     // Path related
     {
+      when: response => {
+        return response.apptype[1] === 'application'
```

Comment:
Nope apptype is an array, so that the user doesn’t have to answer two prompts, this allows for greatly simplified conditionals further down the line.